src/index.spec.ts

```diff
@@ -1,133 +1,17 @@
-import { compose, Next, Middleware } from './index'
+// tslint:disable handle-callback-err
```

Comment:
Can we avoid disabling lint checks? You can pass `done(err)` instead.

Reply:
There is some cases that I want to test but linter prevents me:

An error handler that doesn't propagates the error:
``` javascript
const middleware = compose(
    function (err: Error | null, req: any, res: any, next: Next) {
      req.one = true
      next()
    }
)
```

Reply:
Why can't you write a test? E.g. `expect(err).not.toBe(null)`?

Reply:
The problem is this function: 
```
function (err: Error, req: any, res: any, next: Next) {
    req.one = true
    next()
}
```
Linter says:
```
Expected error to be handled
```
But I want to use this kind of function because I want to test what happens when a middleware doesn't propagate the error to the next error handler.